### DISCUSSION TITLE
How to Solve the error about connecting Euphony Library in Our App?

### DISCUSSION THREAD

**Discussion by win-luck** [euphony-io/euphony#170] [General]
![image](https://user-images.githubusercontent.com/53044069/181014537-7afd6485-a2f1-4749-8658-44f2b5b56021.png)

여러 방면으로 해결책을 찾는 중인데, 혹시 마이크 접근 권한을 주지 않으면 라이브러리에서 클래스를 가져오는 것조차 허용되지 않는 건가요? 현재 마이크 접근 권한 관련 처리 중입니다,,

**Comment by designe** [comment]
마이크 접근 권한 없어도 클래스 가져오는 데는 아무런 문제가 없습니다~!
하지만 앱을 실행할때 권한이 없으면 문제가 발생합니다 :)
